app/public/javascripts/calendar.js

```diff
@@ -119,17 +131,32 @@ var Calendar = function() {
 				}
 				i++;
 			}
-			if (calendar[5]) {
-				for (i = 0; i < calendar[5].length; i++) {
-					if (calendar[5][i] !== "") {
-						calendar[4][i] = "<span>" + calendar[4][i] + "</span><span>" + calendar[5][i] + "</span>";
+			//this function was created shared days if there were more than 5 weeks in a month
+
+			// if (calendar[5]) {
+			// 	for (i = 0; i < calendar[5].length; i++) {
+			// 		if (calendar[5][i] !== "") {
+			// 			calendar[4][i] = "<span>" + calendar[4][i] + "</span><span>" + calendar[5][i] + "</span>";
+			// 		}
+			// 	}
+			// 	console.log("calendar before slice is " + calendar);
+			// 	calendar = calendar.slice(0, 5);
+			// 	console.log("calendar after slice is " + calendar);
+			// }
+			for (i = 0; i < calendar.length; i++) {
```

Comment:
This method is a good candidate for making use of underscore for more terse code.  I would start by taking a look at the each method: http://underscorejs.org/#each